discovery/src/main/java/com/ibm/watson/developer_cloud/discovery/v1/query/Aggregation.java

```diff
@@ -11,7 +11,7 @@
  * specific language governing permissions and limitations under the License.
  */
 
-package com.ibm.watson.developer_cloud.discovery.v1.model.query;
+package com.ibm.watson.developer_cloud.discovery.v1.query;
```

Comment:
are we planning on using these classes or not? seems like the actual service uses `QueryAggregation` instead of this one